components/layout/taffy/stylo_taffy/convert.rs

```diff
@@ -32,30 +33,30 @@ mod stylo {
     pub(crate) use style::values::specified::GenericGridTemplateComponent;
 }
 
+use taffy::MaxTrackSizingFunction;
+use taffy::style_helpers::*;
+
 #[inline]
 pub fn length_percentage(val: &stylo::LengthPercentage) -> taffy::LengthPercentage {
-    if let Some(length) = val.to_length() {
-        taffy::LengthPercentage::Length(length.px())
-    } else if let Some(val) = val.to_percentage() {
-        taffy::LengthPercentage::Percent(val.0)
-    } else {
-        // TODO: Support calc
-        taffy::LengthPercentage::Percent(0.0)
+    match val.unpack() {
+        stylo::UnpackedLengthPercentage::Length(len) => length(len.px()),
+        stylo::UnpackedLengthPercentage::Percentage(percentage) => percent(percentage.0),
+        stylo::UnpackedLengthPercentage::Calc(_) => length(0.0),
```

Comment:
This was `taffy::LengthPercentage::Percent(0.0)`.
Is `length(0.0)` a better fallback than `percent(0.0)`?

Reply:
I've just put it back to what it was for now